src/stylesheets/components/_banner.scss

```diff
@@ -0,0 +1,198 @@
+.usa-banner {
+  background-color: $color-gray-lightest;
+  padding-bottom: 0.7rem;
+
+  @include media($small-screen) {
+    font-size: $h6-font-size;
+    padding-bottom: 0;
+  }
+
+  .usa-accordion-content {
```

Comment:
Is there a reason why we are using `usa-accordion-content` and not `usa-banner-content`?